### PR TITLE
Sending user email in CommentServlet.java 

### PR DESCRIPTION
**Pull request by gracetian6**
- sent user email info as JSON to `script.js` in `CommentServlet.java`
- display email address of user's who post comments via `script.js`
See demo video [here](https://drive.google.com/file/d/1h072AmFLOs3Caon3RlS6vbIwRU6qHT-C/view?usp=sharing)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/User.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.google.sps.data;
 
+
```
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/data/User.java]: nit: remove extra blank line

File: portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java [hunk 2]
```diff
@@ -33,7 +33,8 @@
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+
```
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java]: nit: remove extra blank line above comment

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 3]
```diff
@@ -33,6 +31,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html");
     UserService userService = UserServiceFactory.getUserService();  
 
+
```
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java]: nit: remove extra line

**Review by jessicaslaughter — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java [hunk 4]
```diff
@@ -44,10 +45,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     int numComments = Integer.parseInt(request.getParameter("numComments"));    
-    int count = 0;
     List<Comment> entries = new ArrayList<>();
-
-    UserService userService = UserServiceFactory.getUserService();  
+    UserService userService = UserServiceFactory.getUserService();
+    int count = 0;
```
Comment (jpnickolas) [inline portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java:49]: Is count ever used?

**Review by jpnickolas — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java (hunk 4 above)
Comment (gracetian6) [reply portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java:49]: yes, it's used in the if statement below and incremented before the end of the for loop

Comment (jpnickolas) [reply portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java:49]: Oops, my mistake. Resolving.

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -102,4 +101,4 @@ function createListElement(text) {
   const liElement = document.createElement('li');
   liElement.innerText = text;
   return liElement;
-}
+}
```
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: end file with new line

**Review by jpnickolas — APPROVED** [review]

**Review by jessicaslaughter — APPROVED** [review]

**Pull request merged**